arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -568,6 +547,12 @@ class ffmpeg_video_input::priv
       // Continue to make seek request further back until we land at a frame
       // that is before the requested frame.
       frame_ts -= this->f_backstep_size * this->stream_time_base_to_frame();
+      num_of_attempts++;
+      if ( num_of_attempts > this->max_seek_back_attempts )
```

Comment:
```suggestion
      if ( ++num_of_attempts > this->max_seek_back_attempts )
```

It shouldn't matter for a POD type where the expression result is unused, but just on principle I always advocate preferring pre-increment unless you actually *need* post-increment.